bsd.go

```diff
@@ -7,23 +7,23 @@ package gopass
 #include <unistd.h>
 #include <stdio.h>
 
-int getch() {
-        int ch;
+char getch(int termDescriptor) {
+        char ch;
         struct termios t_old, t_new;
 
-        tcgetattr(STDIN_FILENO, &t_old);
+        tcgetattr(termDescriptor, &t_old);
         t_new = t_old;
         t_new.c_lflag &= ~(ICANON | ECHO);
-        tcsetattr(STDIN_FILENO, TCSANOW, &t_new);
+        tcsetattr(termDescriptor, TCSANOW, &t_new);
 
-        ch = getchar();
+        read(termDescriptor, &ch, sizeof(ch));
```

Comment:
This is the main change in the latest commit. Also the whole function returns `char` instead of `int`. (If I read a single byte into a `sizeof(int)` buffer, I'll run into problems on big-endian platforms, right?)